### DISCUSSION TITLE
How do I get one?

### DISCUSSION THREAD

**Discussion by felipeholanda2077** [community/community#30998] [Other Feature Feedback, Questions, & Ideas]
I would like to know how to receive for my profile

**Comment by radar-blip** [comment]
If you are talking about achievement badges, look here: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by felipeholanda2077** [reply]
thank you very much I will take a look at the content, that's about it

**Reply by radar-blip** [reply]
You're welcome!

**Comment by MdHRShohel** [comment]
if you get your answer then mark this message as a answered
